.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
     - CMAKE_CXX_FLAGS="-DCMAKE_CXX_FLAGS_RELEASE=-O3 -g"
     - USE_LOG4CPP=-DUSE_LOG4CPP=TRUE
     - USE_ROCKSDB=-DBUILD_ROCKSDB_STORAGE=TRUE
-      #- USE_S3_OBJECT_STORE=-DUSE_S3_OBJECT_STORE=FALSE
+    - USE_S3_OBJECT_STORE=-DUSE_S3_OBJECT_STORE=OFF
```

Comment:
`-DUSE_S3_OBJECT_STORE=ON` - see my previous comment

Reply:
Okay, this is also a good approach. Let's see if they will pass this way.